bundles/org.openhab.binding.energenie/README.md

```diff
@@ -0,0 +1,97 @@
+# Gembird energenie Binding
+
+This binding integrates the Gembird energenie range of power extenders by using the Energenie Data Exchange Protocol and power reading devices through HTTP interface. 
+
+
+## Supported Things
+
+The Binding supports PM2-LAN, PMS-LAN, PMS2-LAN or PMS-WLAN power extenders as well as PWM-LAN power measurement devices.
+
+## Discovery
+
+Gembird energenie devices don't support autodiscovery.
+All Things need to be created manually either in PaperUI or within Things files.
+
+
+## Binding Configuration
+
+The Binding does not need any specific configuration
+
+## Thing Configuration
+
+The device requires the IP-address and a password as a configuration value in order for the binding to know where to access it and to login to the device. 
+
+| Parameter | Description                                          |
+|-----------|------------------------------------------------------|
+| host      | IP-Address of energenie device                       |

```

Comment:
Are you sure it only works with an IP address? If I'm not mistaken, a host name should also be ok. If this is true, you could adjust the documentation and the description in the `thing-type.xml`

Reply:
As the original App also just uses IP-Adress (does a complete address range ping for discovery), I would not want to change this. Even I cannot test this for PWM devices, cause not having those at hands.